src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -357,8 +357,8 @@ trait FastSync {
       val (nonMptNodesToGet, remainingNonMptNodes) = nonMptNodesQueue.splitAt(nodesPerRequest)
       val (mptNodesToGet, remainingMptNodes) = mptNodesQueue.splitAt(nodesPerRequest - nonMptNodesToGet.size)
       val nodesToGet = nonMptNodesToGet ++ mptNodesToGet
-      val handler = context.actorOf(
-        FastSyncNodesRequestHandler.props(peer, etcPeerManager, peerEventBus, nodesToGet, blockchain, blockchainStorages.mptNodeStorage))
+      val handler = context.actorOf(FastSyncNodesRequestHandler.props(peer, etcPeerManager, peerEventBus, nodesToGet, blockchain,
+        blockchainStorages.nodesKeyValueStorageFor(Some(targetBlockOffset))))
```

Comment:
If I understood correctly the parameter of `nodesKeyValueStorageFor` should be the block number to which the nodes stored belong, should then it be the target block instead? (which I think is `Some(initialSyncState.targetBlock)`)